corva/middleware/loader.py

```diff
@@ -0,0 +1,14 @@
+from typing import Callable, Optional
+
+from corva.models.base import BaseContext
+
+
+def loader_factory(loader: Callable, loader_kwargs: Optional[dict] = None) -> Callable:
```

Comment:
let's call this `get_loader` or `get_loader_fn`. It's basically what this function does. Also, it's not clear for me what loader does

Reply:
it will be deleted in pr #14

Reply:
why do we review functionality which does not exist in the next PR? It does not make sense

Reply:
branch in pr #14 was created from this one, so it had this functionality. it just turned out in process of development of pr #14 that we can get rid of this file